packages/menu-bar/src/vaadin-menu-bar-buttons-mixin.js

```diff
@@ -28,6 +28,16 @@ export const ButtonsMixin = (superClass) =>
       return ['_menuItemsChanged(items, items.splices)'];
     }
 
+    constructor() {
+      super();
+
+      this.__parentResizeObserver = new ResizeObserver(() => {
```

Comment:
Would it make sense to move this to `resize-mixin` as an opt-in feature?

Reply:
Yes, sounds like a good idea. I will create a separate PR for that.